packages/block-library/src/image/edit.native.js

```diff
@@ -205,17 +205,9 @@ class ImageEdit extends React.Component {
 	}
 
 	render() {
-		const { attributes, isSelected } = this.props;
+		const { attributes, isSelected, onSettingsToggle } = this.props;
 		const { url, height, width, alt, href, id } = attributes;
 
-		const onImageSettingsButtonPressed = () => {
-			this.setState( { showSettings: true } );
-		};
-
-		const onImageSettingsClose = () => {
-			this.setState( { showSettings: false } );
-		};
-
```

Comment:
move `BottomSheet` open/close handler to the top (`block.native.js`) as well as `showOptions` prop of the state